backend/server/src/db/migrations/5-events.sql

```diff
@@ -0,0 +1,15 @@
+call prepare_randomized_ids('event');
```

Comment:
```suggestion
call prepare_randomized_ids('events');
```

In `series`, the identifier is called the same as the table.

Reply:
Weeeeell, the table is called `plural(entity)` and the generation stuff is called `singular(entity)`, which happens to be the same for series, but not for events. This doesn't really matter besides aesthetics, of course, but this call would create a `sequence` called `__interna_events_ids`. I think that double plural sounds weird. :thinking:

Reply:
Ah yeah ok fair.